aten/src/ATen/TensorMeta.h

```diff
@@ -62,14 +77,58 @@ namespace impl {
 //
 // A notable subclass of this interface is TensorIteratorBase.
 struct TORCH_API MetaBase {
-  virtual void set_output(int64_t output_idx, IntArrayRef sizes, IntArrayRef strides, TensorOptions options, DimnameList names) = 0;
+  virtual void set_output(int64_t output_idx, IntArrayRef sizes, IntArrayRef strides, TensorOptions options, DimnameList names) {
+    set_output_raw_strided(output_idx, sizes, strides, options, names);
+  }
+
   virtual const Tensor& maybe_get_output(int64_t output_idx) = 0;
   void set_output(IntArrayRef sizes, TensorOptions options) {
     set_output(0, sizes, {}, options, {});
   }
   void set_output(int64_t output_idx, IntArrayRef sizes, TensorOptions options) {
     set_output(output_idx, sizes, {}, options, {});
   }
+
+  // See: https://github.com/pytorch/pytorch/issues/69813
```

Comment:
I feel the documentation here is not clear enough, people will not understand how to use this API correctly just by reading these comments. I think adding some sort of flow chart of how to decide pick which one will help.